src/applications/personalization/dashboard-2/components/health-care/Appointments.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { format } from 'date-fns';
+import moment from 'moment';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 import CTALink from '../CTALink';
 
 export const Appointments = ({ appointments, hasError }) => {
   const nextAppointment = appointments?.[0];
-  const start = new Date(nextAppointment?.startsAt);
+  const start = moment.parseZone(nextAppointment?.startsAt);
```

Comment:
This is the third major change. Use `parseZone()` to make sure the date is presented correctly, in the time zone where the appointment takes place, rather than the timezone of the user's browser.